packages/cli/src/TestWebhooks.ts

```diff
@@ -177,6 +175,21 @@ export class TestWebhooks implements IWebhookManager {
 		return webhookMethods;
 	}
 
+	async findAccessControlOptions(path: string, httpMethod: IHttpRequestMethods) {
+		const webhookKey = Object.keys(this.testWebhookData).find(
+			(key) => key.includes(path) && key.startsWith(httpMethod),
+		);
+
+		const nodes = webhookKey ? this.testWebhookData[webhookKey].workflow.nodes : {};
+		const webhookNode = Object.values(nodes).find(
+			({ type, parameters }) =>
+				type === 'n8n-nodes-base.webhook' &&
```

Comment:
The check for a specific node type means this would not work for generic nodes such as `GithubTriggerNode`, right?